frontend/src/entry/model/InferenceServices.tsx

```diff
@@ -27,6 +46,32 @@ export default function InferenceServices({ model }: InferenceProps) {
     [model, inferences],
   )
 
+  const handleCreateToken = async () => {
+    const error = MultipleErrorWrapper('Unable to create an authenication token', { isTokensError, isUiConfigError })
+    if (error) return error
```

Comment:
Instead of returning jsx here, perhaps use `setErrorMessage` with a string.

Reply:
This wraps mulitple errors within a jsx element in the case that both of them fail.